ui/desktop/electron-app/src/models/runtime-settings.js

```diff
@@ -38,27 +38,27 @@ class RuntimeSettings {
         throw new Error();
       }
     } catch (e) {
-      throw new Error(`Origin ${origin} could not be validated.`);
+      throw new Error(`Cluster URL ${clusterUrl} could not be validated.`);
     }
   }
 
   /**
-   * Sets the origin to null.
+   * Sets the clusterUrl to null.
    */
-  async resetOrigin() {
-    this.origin = null;
```

Comment:
@hashicorp/boundary-ui I think this is a bug we fixed with this refactor.

Explanation:
- Within this file, the internal private var is declared `#origin`.
- Within all the methods that work or change that var is used as `#origin` but except this method, that was referencing it as `origin` (without the `#`).

On the refactor I changed this to add the `#` to the variable, which I think fixes a bug when reseting the cluster URL (former origin).

Thoughts?

Reply:
I actually don't think this is a bug. It looks like it's using the setter to set the variable instead of directly accessing the private field. It depends if we want the `triggerOriginChanged` function to run when reset is called.

FYI I included a fiddle to show the private field setter difference for anyone who hasn't used this yet in JS: https://jsfiddle.net/6stq21k7/

Reply:
A-W-E-S-O-M-E @ZedLi I didn't though about that on that way and you are right. I though we were trying to access directly there. Thank you!!